src/components/PodcastListCtrl/PodcastListCtrl.tsx

```diff
@@ -392,7 +392,7 @@ class PodcastListCtrl extends Component<Props, State> {
           dataPodcast={x}
           handleLinkClick={this.linkClick}
           hasLink={true}
-          itemType='podcast'
+          itemType={PV.query.podcast}
```

Comment:
This itemType attribute is actually different than PV.query.podcast.

PV.query.podcast represents the URL path ending in the query.

itemType is a custom attribute on this MediaListItem component. There's no guarantee that the URL path will match the itemType attribute, so they need to be 2 separate constants.

Also, the MediaListItem is a mess. It's one of the first components I wrote for the site 2 years ago and it's total garbage :[